deployments/dev/values.yaml

```diff
@@ -0,0 +1,22 @@
+global:
+  domain:
+    prefix: dev
+app:
+  image:
+    tag: "4.3.0-rc.1-app"
```

Comment:
these can follow any of the image pattern, but just so we know we are testing the right thing matched with staging.